### PR TITLE
Check-in issues resolution + plotting

### LINKED ISSUE #12: Add nbfi_use and informal_group_use

### LINKED ISSUE #21: Farming map to include fishing
Add DL4_1 (income from livestock, fish, poultry) etc to 'farm' definition for percent farming map

### LINKED ISSUE #30: Regression coefficient boxplots

### LINKED ISSUE #34: Re-run bokeh plot w/ updated MFI vars

### PR DESCRIPTION
**Pull request by ejm714**
- Resolves final issues from check-in.
- Splits modeling notebook into draft notebook (3.0.1 which is a playground of different models that will be refactored later) and a plotting notebook that calculates f1 scores for a range of undersampling ratios and then plots coefficients in a boxplot and bar chart

Note: this updated the mmr_analysis_frame.csv which has been synced with s3

Closes #21 
Closes #34 
Closes #12 
Closes #30

### CODE REVIEW

File: notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+        new_labs = []
+        for t in ax.get_yticklabels():
+            text = t.get_text()
+
+            if '(' in text:
+                t.set_text(text[:text.index('(')])
+                new_labs.append(t)
+            else:
+                new_labs.append(t)
```
Comment (pjbull) [inline notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py]: I know this whole block is just what we had in the old NB, but I tihnk it's not necessary for these plots

File: notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+vif["vif_factor"] = [variance_inflation_factor(X_resampled, i) for i in range(X_resampled.shape[1])]
+vif["features"] = dummied.columns
+
+vif.sort_values(by='vif_factor')
+
+
+# In[44]:
+
+
+ratio_list = [5000, 10000, 13000, 17000, 20000, 30000, 50000, 70000, 100000, 150000]
```
Comment (pjbull) [inline notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py]: For my own knowledge, are these ratios or absolute number of observations?

File: notebooks/2.4-ejm-mmr-bokeh-heatmap/2.4-ejm-mmr-bokeh-heatmap.py [hunk 3]
```diff
@@ -38,7 +38,7 @@
 analysis_frame = pd.read_csv(DATA_PATH.parent/'processed'/'mmr_analysis_frame.csv', index_col=0)
 
 
-# In[3]:
+# In[8]:
```
[the remark this answers is not in the document]
Comment (ejm714) [reply notebooks/2.4-ejm-mmr-bokeh-heatmap/2.4-ejm-mmr-bokeh-heatmap.py]: The thing that changed was the Bokeh heatmap output which was re-run w/ the updated analysis frame and I wanted that to show that in the latest version of the notebook. Though you're totally right there weren't any changes to the .py file so will watch that in the future 👍

File: notebooks/2.0.1-ejm-analysis-prep-models-myanmar/2.0.1-ejm-analysis-prep-models-myanmar.py [hunk 4]
```diff
@@ -304,6 +304,7 @@ def _fill_income(var, label):
 })
 
 features.append('literacy')
+ordinals.append('literacy')
```
[the remark this answers is not in the document]
Comment (ejm714) [reply notebooks/2.0.1-ejm-analysis-prep-models-myanmar/2.0.1-ejm-analysis-prep-models-myanmar.py]: Done

File: notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+#             clf_resampled.fit(X_resampled, y_resampled)
+#             f1_rounded = lambda est, X, y: f1_score(y, np.round(est.predict(X)), average='weighted')
+#             f1 = np.mean(cross_val_score(clf_resampled, X_resampled, y_resampled, cv=5, scoring = (f1_rounded)))
+            
+        cdf['r_'+ str(r)] = pd.Series(dict(zip(dummied.columns, clf_resampled.coef_.flatten())))
+        d[r] =  f1 
+                 
+    cdf = cdf.reindex_axis(cdf.mean(axis=1).sort_values(ascending=False).index, axis=0).transpose()
+            
+    return cdf, d
```
[the remark this answers is not in the document]
Comment (ejm714) [reply notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py]: It's actually more helpful as a dictionary as it allows us to look up the ratios we want to use in our plots based on their f1_scores

File: notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py (hunk 1 above)
Comment (ejm714) [reply notebooks/3.0.1-ejm-scatter-boxplot-bar/3.0.1-ejm-scatter-boxplot-bar.py]: True story. Removed

**Comment by ejm714** [comment]
@pjbull Removed the extra code from the bar plot function so should be good to go now.

**Pull request merged**